src/m365/cli/commands/config/config-set.spec.ts

```diff
@@ -30,6 +30,10 @@ describe(commands.CONFIG_SET, () => {
     Utils.restore(Cli.getInstance().config.set);
   });
 
+  afterEach(() =>{
+    Utils.restore(Cli.getInstance().config.set);
```

Comment:
If we reset the config after each test, we don't need to reset it after all tests (above) and can remove it there.